lib/nodejs/vwf.js

```diff
@@ -139,19 +141,37 @@ function HandleFileRequest( request, response ) {
 // directory structure.
 // If all that fails, serve up a 404 response since the request was not handled.
 function Serve( request, response ) {
+    lobby.post( "/sim", function( req, res ) {
```

Comment:
@davideaster I forgot that you said that I should move this into `lobby.js`.  I can do that in a future PR (unless I get to it before you review this).  :wink: